feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -0,0 +1,169 @@
+                    bottom = sizes.smallerSize
+                )
+        ) {
+            Icon(
+                Icons.Outlined.Phone,
+                contentDescription = stringResource(id = R.string.phone_icon),
+                modifier = Modifier.padding(end = sizes.minorSmallSize),
+                tint = MaterialTheme.colorScheme.tertiary
+            )
+            Text(
+                text = "(404) 979-2400",
+                fontWeight = FontWeight.Normal,
+                color = MaterialTheme.colorScheme.tertiary
+            )
+        }
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = sizes.regularSize, vertical = sizes.smallerSize)
+        ) {
+            Icon(
+                Icons.Outlined.Mail,
+                contentDescription = stringResource(id = R.string.mail_icon),
+                modifier = Modifier.padding(end = sizes.minorSmallSize),
+                tint = MaterialTheme.colorScheme.tertiary
+            )
+            Text(
+                text = "juan.islas@mail.com",
+                fontWeight = FontWeight.Normal,
+                color = MaterialTheme.colorScheme.tertiary
+            )
+        }
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(
+                    start = sizes.regularSize,
+                    end = sizes.regularSize,
+                    top = sizes.smallerSize,
+                    bottom = sizes.smallSize
+                )
+        ) {
+            Icon(
+                Icons.Outlined.LocationOn,
+                contentDescription = stringResource(id = R.string.location_icon),
+                modifier = Modifier.padding(end = sizes.minorSmallSize),
+                tint = MaterialTheme.colorScheme.tertiary
+            )
+            Text(
+                text = "4950 Gaidrew, Alpharetta, GA, 30022",
+                fontWeight = FontWeight.Normal,
+                color = MaterialTheme.colorScheme.tertiary
+            )
+        }
+        Divider(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = sizes.regularSize, vertical = sizes.midSmallSize),
+            color = MaterialTheme.colorScheme.onPrimary
+        )
+        LpOutlinedButton(
+            modifier = Modifier.padding(horizontal = sizes.smallSize),
+            textButton = stringResource(id = R.string.payment_methods),
+            onClick = { /*TODO high order function for payment methods*/ })
+        LpOutlinedButton(
+            modifier = Modifier.padding(horizontal = sizes.smallSize, vertical = sizes.nothingSize),
+            textButton = stringResource(id = R.string.emergency_contacts),
+            onClick = { /*TODO high order function for emergency contacts*/ })
+        LpOutlinedButton(
+            modifier = Modifier.padding(horizontal = sizes.smallSize, vertical = sizes.nothingSize),
```

Comment:
Please check the paddigns between the buttons because is  not same as the design

Reply:
![image](https://user-images.githubusercontent.com/58410224/232540881-4c16f19b-422f-4e78-92ee-b057e0837d1a.png)

I had to modified LpOutlinedButton padding directly, by default it has a padding of 10.dp, I changed for 0.dp, I do not know if I could do that.